src/search/engines/_sedefaults/fields.go

```diff
@@ -0,0 +1,61 @@
+package _sedefaults
+
+import (
+	"strings"
+
+	"github.com/PuerkitoBio/goquery"
+	"github.com/hearchco/hearchco/src/search/engines"
+	"github.com/hearchco/hearchco/src/search/parse"
+	"github.com/rs/zerolog/log"
+)
+
+// Fetches from DOM via dompaths. Returns (url, title, description)
+func RawFieldsFromDOM(dom *goquery.Selection, dompaths *engines.DOMPaths, seName engines.Name) (string, string, string) {
```

Comment:
does pointer for goquery.Selection and engines.DOMPaths bring any noticable performance benefits? if not, better to copy since it assures no changes when returned

Reply:
The dom (goquery.Selection) is given as a pointer by colly. The dompaths could be passed by value, but since this function is called on every result (for most engines), its probably best to keep it pointer (it would be consistent with the codebase currently, per-result structs are passed by pointer). I will add a comment to the function though.

Reply:
Dompaths is just a few strings, you're already making more strings using `dom.Find`. All of that gets cleaned up quickly since it's on stack. Don't see a point making it a pointer, only allows for bugs to crawl up in the future.